inst/extdata/abm.Rmd

```diff
@@ -133,7 +138,6 @@ if(df_cf %>% filter(!is.na(P_frac)) %>% nrow() > 0){
   
   grid.arrange(pub_bullet1, pub_bullet2, ncol = 1)
```

Comment:
patchwork? https://www.datanovia.com/en/blog/ggplot-multiple-plots-made-ridiculuous-simple-using-patchwork-r-package/

Reply:
Very interesting package, thanks for link!

Reply:
this was mostly an idea for future refactoring...